### ISSUE TITLE
Feedback for ModDB

### ISSUE THREAD

**Issue by MattFiler** [DBolical/Home#49]
I posted this originally in the ModDB feedback forum and was asked to repost here, so here it is!

After using the site for quite a while now and hosting a few successful mods there are some things that I'd definitely like to see improved (but don't get me wrong, it's an amazing platform and one of the very few sites I have whitelisted on AdBlock).

I'd like an easier way to contact the staff. Getting support is quite annoying, and the easiest way is to contact staff themselves even though I'm sure they probably don't like you doing that. If there was an official "contact us" form that was actually monitored often (it took ages for me to get a response from the current one) it would go a long way to building links to the community.

What would also be welcome is a more user friendly design that's also responsive so it is easy to view on mobile. The current layout is almost 10 years old I believe, and it really shows. Fair enough it's a big undertaking to redesign the entire site, but it's something that would draw a lot more people in.

An easier to use notification system would be great. Currently my notifications are building up and up and it would be great if there was a similar system to Google, so you have a notification icon and a notification count, you click on that and can see all your unread notifications, then once you've looked at them, the count goes back to zero. Similar to the system in place with messages at the moment.

The messaging system also could be made a bit more fluid, having to scroll down a long page of messages every time I load up a conversation is really frustrating, it would be good if you only displayed, for example, the last 10 messages and then you had the option to load more. Some kind of instant reply system would also be good, so instead of having to keep refreshing the page, there's some kind of AJAX being used that does it for you to keep the chat updated.

Make mod pages more like Steam! It would be great to just have simply a slider full of images about the mod, a block of text about the mod, a logo and a download button. Reviews underneath. It would be so much better to have like proper product pages and would help to make the site feel more like a distribution platform than a file hosting site with comments.

Get rid of the archiving system from news posts, or find a way to improve it. So many devs are put off sending out updates because they know that if they don't put at least half an hours work into it sourcing new images and videos then the post will be archived and they may not of bothered, this leads to projects becoming stale and people constantly asking "Is this dead?". If you allowed simple short updates on projects, developers would be way more encouraged to post news for fans to try and keep them engaged with what's happening. The delay on news posts is also quite frustrating, sometimes news needs to get out fast, and having to wait 5-6 hours (maybe longer) to get a post verified just really slows everything down.

Finally, please improve the search feature of this site. It's way too restrictive. For example, if I want to find Alien Isolation mods, I have to actually type in the correct formatting of the game's title "Alien: Isolation" else it won't show up until I go into the actual full results page. It would be so much easier if it was more lenient on finding games/mods/developers.

There are a few other issues, but I think those are my main issues with the site. It's a great service though and one I use every day, not just for monitoring my own projects but for checking out other people's work too.

**Comment by feillyne** [comment]
> Get rid of the archiving system from news posts, or find a way to improve it. So many devs are put off sending out updates because they know that if they don't put at least half an hours work into it sourcing new images and videos then the post will be archived and they may not of bothered, this leads to projects becoming stale and people constantly asking "Is this dead?". If you allowed simple short updates on projects, developers would be way more encouraged to post news for fans to try and keep them engaged with what's happening. The delay on news posts is also quite frustrating, sometimes news needs to get out fast, and having to wait 5-6 hours (maybe longer) to get a post verified just really slows everything down.

No, the current archiving system for articles is needed because there would be too many developers pushing their news with bad content; too short articles, too few images, posts in another language, etc. (Also, AFAIK, you can still mark your post as archived/private by default and bypass authorisation/waiting process.) Sadly or not, the front page is only for relevant, good posts.

What would be nice to see, is a **_3-tier system_** for articles: authorised, archived and private articles, but both authorised (featured) and archived posts would be visible on the front page - authorised articles shown by default, and archived posts displayed after user clicked "archived articles" tab, while private posts could not be displayed anywhere except on the linked pages (just like it is now for archived posts).

So the best, highest quality articles would go under 1) Authorised category, relevant posts lacking quality under 2) Archived, and irrelevant (to development, gaming, the site itself such as those posted under hobby groups) posts would go under 3) Private label. To iterate, both authorised and archived posts would be displayed on the front page albeit archived ones would be accessible only through a togglable button.

The 3-tier system (authorised/archived/private or featured/authorised/archived system) would be much better than the current 2-tier system (authorised/archived) IMHO, as there is no middle ground now.
